styles/standard.mapcss

```diff
@@ -155,6 +155,11 @@ way|z11-[railway=razed][!bridge][!tunnel]["razed:ref"]
 {
 	text: eval(join(" ", tag("razed:ref"), tag("razed:name")));
 }
+way|z11-[railway=dismantled][!bridge][!tunnel]["dismantled:name"],
```

Comment:
I don't think this is worth it. There are as of today 6 ways with dismantled:name and none with dismantled:ref in taginfo.